toolkit/tools/imageconfigvalidator/imageconfigvalidator_test.go

```diff
@@ -90,5 +100,6 @@ func TestShouldFailDeeplyNestedParsingError(t *testing.T) {
 			return
 		}
 	}
-	assert.Fail(t, "Could not find 'core-efi.json' to test")
+	assert.Fail(t, "Could not find "+targetPackage+" to test")
```

Comment:
>assert.Fail(t, "Could not find "+targetPackage+" to test") [](start = 1, length = 58)

Nit: I think there was an `assert.Failf` we could use to follow the same way we format strings elsewhere.

Reply:
Bit tougher to use the Failf version since you need to pass both an error string (not formattable), and the formattable message. Updated however since I have used it in a few other spots.

---
In reply to: [549908766](https://github.com/microsoft/CBL-Mariner/pull/480#discussion_r549908766) [](ancestors = 549908766)